mongo/tests/test_unit.py

```diff
@@ -2,6 +2,7 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import copy
+import urllib
```

Comment:
nit:

```suggestion
from urllib.parse import quote_plus
```

Reply:
I guess this implies to remove `urllib.parse.` from calls too